### DISCUSSION TITLE
Welcome to manage-landing-page-master Discussions!

### DISCUSSION THREAD

**Discussion by Walter-Tronics** [Programmers-Hub-International/manage-landing-page-master#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Walter-Tronics** [comment]
@wizzy-design 
@kingchi005 
@Syipmong 
@marviecephas 
@TheAce74 
@OtiKelsy7 
@TomiwaAborowa 
@devnasfam

**Comment by Syipmong** [comment]
Ok thanks for adding me

**Comment by TheAce74** [comment]
I'm happy to be here

**Comment by Walter-Tronics** [comment]
Who's in the desktop view part also, please signify. So we can move on to assigning tasks.

**Comment by Syipmong** [comment]
Can I have a briefing on what's happening because I am totally lost😑

**Reply by Walter-Tronics** [reply]
We spoke about this in the WhatsApp group. But again, we are working on a template to design as a team. Desktop and mobile view. Since the desktop has already started, we go through completing the work and then pass it on to the mobile responsiveness. Over here, tasks can be assigned to anyone to work on a section so we can avoid conflicts like working on the same button tag and overriding styles unnecessarily. Please check my recent comment on this discussion forum to see what was said. Also you can check the "project" section of the landing page repository to see progress. At the baseline, we should all commit to this discussion section. It's through here things will be discussed and tasks given. Kindly add other members from the group to the community. I hope you understand.

**Comment by wizzy-design** [comment]
Sup guys

**Comment by Syipmong** [comment]
Boss Wiss👋

**Comment by Syipmong** [comment]
Sure, will do that.

**Comment by Walter-Tronics** [comment]
Okay, guys, the desktop view has been finalized, you can check it with the link in the repo. Now we move to the mobile view.
Those with the mobile team should please signify.
@TheAce74 
@TomiwaAborowa 
@wizzy-design 
@devnasfam 
@Einsteinj04 
@Syipmong 
@kingchi005 
@OtiKelsy7 
@Paulcode405 
@marviecephas

**Reply by TheAce74** [reply]
Good morning guys, how do we go about the mobile view, do we split it into separate major components then select people to handle each major component?
Cus I'm seeing 6 different sections (header, hero, about, testimonial, cta and footer)
They're 10 of us in the mobile so I'm guessing 8 to 4 sections and the other 2 will handle 2 sections
I suggest following the order of sections in handling them so as to avoid using many media query declarations

**Reply by Walter-Tronics** [reply]
I think it's just 8 people now for mobile. And yeah I support what you said.
Others should comment below so everyone knows where they are.

**Reply by Paulcode405** [reply]
I'm here boss, please I really don't know where I'll fall in... I know just HTML, CSS and plain JS no knowledge on react, framework or the rest for now

**Reply by Walter-Tronics** [reply]
All we're using is Vanilla everything. Plain or core languages. It's in the discription

**Comment by Syipmong** [comment]
Yes, I agree.

**Comment by Syipmong** [comment]
Thank you sir

**Comment by wizzy-design** [comment]
I'd really like to contribute to this project, but right now my school cut us (boys hostel 😢) from wifi and that's the only way i can come online. Sorry. Until they fix it i can't ba easily contacted. I'm in school building rn. That's why i can talk

**Comment by Syipmong** [comment]
You don't need to worry about that
We got you 💯 sir

**Comment by OtiKelsy7** [comment]
Is there any progress yet?

**Reply by wizzy-design** [reply]
If i may ask, asides me and @TomiwaAborowa (2 people). Which other people contributed to the desktop version ? I want to know how many people did so

**Reply by Walter-Tronics** [reply]
Please feel free to check it, it's there. GitHub is powerful enough to provide you with that information.
But I am the one who finished up the desktop version from where you guys stopped.
